\documentclass[11pt, letterpaper, titlepage]{article}
\usepackage[utf8]{inputenc}
\usepackage[export]{adjustbox}
\usepackage{geometry}
 \geometry{
 a4paper,
 total={168mm,257mm},
 left=15mm,
 right = 15mm,
 top=15mm,
 bottom=15mm,
 includefoot,includehead
 }

\usepackage{siunitx}
\usepackage[backend=biber, style=authoryear, giveninits=true, maxbibnames=25, uniquename=init, maxcitenames=2, hyperref=true, dashed=false]{biblatex}			% Benutze Biber/BibLaTeX zum Zitieren
\addbibresource{main.bib}					% Pfad zur BibTeX Datei aus Citavi
\renewcommand{\cite}{\parencite}
\usepackage{caption}
\usepackage{subcaption}
\usepackage{graphicx}
\usepackage{svg}
\usepackage{placeins}
\usepackage[hidelinks]{hyperref}
\usepackage{amsmath}
\usepackage[headsepline]{scrlayer-scrpage}
\usepackage{acronym}
\usepackage{tabularray} % Selina
%needed for Diffdock images
\usepackage{subcaption}
\usepackage[labelformat=parens,labelsep=quad,skip=3pt]{caption}

\clearpairofpagestyles %Seitenzahl nicht in der Kopfzeile

\title{MeetEU Project - Team Heidelberg - Team 1 -- \\ Identification and Enhancement of novel Sars-CoV-2 NSP13 Helicase Inhibitors}
\author{Linda Blaier, Paul Brunner, Selina Ernst, Valerie Segatz, and Chlo\'{e} Weiler}
\date{February 2024}

\DeclareUnicodeCharacter{2009}{\,}

\begin{document}

\maketitle

\ihead{\headmark}
\automark{section}  %Kopfzeile gleich dem Sektiontitel
\cfoot{\pagemark}   %\ofood Seitenzahl rechts



\section{Abstract}
%Although the development of vaccines against Sars-CoV-2 was successful during the recent pandemic, the amount of FDA approved drugs for the therapy of Covid-19 is still limited to Paxlovid and Veklury, Olumiant and Actemra \cite{FDACOVID}. One possibility to accelerate the development of new therapies for Covid19 is to screen already approved drugs for effects against the viral reproduction. In this years MeetEU project, we investigated the NSP13 helicase of Sars-CoV-2 and tried to find compounds that could be repurposed for this therapy, as well as novel compounds that could lead to an effective treatment of Covid19. Using our \textit{in-silico} pipeline (see graphical abstract) enables us to evaluate possible drug candidates, suggest novel structures based on already approved drugs and investigate their toxicity, while being cheaper and less labor intensive than projects limited to wet-lab work.

The development of innovative therapeutics is essential for preventing another COVID-19 pandemic. Consequently, researchers aim to identify anti-SARS-CoV-2 agents as substitutes for vaccines or immune therapeutics that are able to maintain their efficacy despite the virus's high mutation rate. Targeting the NSP13 helicase of SARS-CoV-2 is a promising strategy due to the protein's high sequence conservation and indispensability for viral proliferation as part of the replication and transcription complex \cite{Marecki,Malone_2022}. Therefore, this project focused on the screening of potential drugs that inhibit the NSP13 helicase by trying to find novel compounds as well as FDA-approved compounds that could be repurposed for an effective treatment of COVID-19. In summary, our proposed \textit{in silico} pipeline, depicted in the graphical abstract, identified the ATP-binding pocket of NSP13 as the consensus binding site. Notably, we identified angiotensin 1-7 as our top scoring ligand, which has a low predicted toxicity and binds stably inside the binding pocket. This finding suggests the potential of the identified protein as an inhibitor of NSP13, warranting further \textit{in vitro} validation.
\FloatBarrier

\begin{figure}[h]
  \centering
  \includegraphics[width=\textwidth]{Workflow_MeetEU.pdf}
  \caption*{Proposed workflow for the discovery of NSP13 helicase inhibitors.}
  \label{workflow}
\end{figure}

\setcounter{figure}{0}
\renewcommand{\thefigure}{\arabic{figure}}
\newpage

%    Abkürzungsverzeichnis
{\setlength{\parskip}{0.2cm}
\section*{Abbreviations}
    \begin{acronym}[LC-MS/MS23]
        % A B C D E F G H I J K L M N O P Q R S T U V W X Y Z        
        % Abkürzungen
    \acro{ADP}{adenosine diphosphate}
    \acro{ATP}{adenosine triphosphate}
    \acro{COVID-19}{coronavirus disease 2019}
    \acro{DScore}{druggability score}
    \acro{FDA}{food and drug administration}
    \acro{MD}{molecular dynamics}
    \acro{MM-PBSA}{molecular mechanics energies combined with the Poisson-Boltzmann and surface area continuum solvation}
    \acro{NSP13}{non-structural protein 13}
    \acro{RAS}{renin-angiotensin system}
    \acro{RCSB PDB}{Research Collaboratory for Structural Bioinformatics Protein Data Bank}
    \acro{RMSD}{root-mean-square deviation}
    \acro{RMSF}{root-mean-square fluctuation}
    \acro{RTC}{replication and transcription complex}
    \acro{SARS-CoV-2}{severe acute respiratory syndrome coronavirus type 2}
    \acro{SAscore}{synthetic accessibility score}
    \acro{ssRNA}{single-stranded RNA}
    \acro{Vina}{AutoDock Vina}
    \acro{ZBD}{zinc-binding domain}
       
        % Formelzeichen
        
        
        % als benutzt markierte Acronyme    
        
        
    \end{acronym}
}
\newpage

\section{Introduction}
In the wake of the successful development of vaccines against \ac{SARS-CoV-2} during the recent pandemic, the amount of FDA-approved drugs for the therapy of \ac{COVID-19} remain limited to Paxlovid, Veklury, Olumiant, and Actemra \cite{FDACOVID}. Expanding the repertoire of drugs available for treating \ac{COVID-19} is particularly crucial for high-risk individuals, as vaccines may not fully prevent infections. The goal of this year's Meet-EU project is to develop a pipeline for identifying potential inhibitors targeting the SARS-CoV-2 helicase known as \ac{NSP13}.
This protein is considered a promising drug target for two reasons. Firstly, \ac{NSP13} is part of the \ac{RTC} which is essential for viral RNA synthesis \cite{Malone_2022}. Secondly, its high sequential and structural conservation across coronaviruses means that there is a reduced likelihood of the virus developing resistances to drugs targeting \ac{NSP13} through mutations of the viral genome \cite{Spratt_2021}. Consequently, effective \ac{NSP13} inhibition would significantly impede viral replication and therefore its spread within the host. 
\ac{NSP13} consists of five domains, namely the \ac{ZBD} with the accessory 2B domain, the $\alpha$-helical stalk domain, as well as the RecA-like 1A and 2A domains \cite{Marecki}. The gap between the RecA-like domains and the 2B domain encompass the site for ATP and RNA binding \cite{NSP13_basics}. In the \ac{RTC} the NSP13 helicase is present as a homodimer. Nevertheless, only one of the copies complexes with a single-stranded RNA \ac{ssRNA}. Therefore, previous publications proposed that the monomer is catalytically active form of the \ac{NSP13} helicase \cite{Berta_2021}. \\

\noindent Computer-aided, structure-based drug discovery is employed for identification of known drugs for their potential as inhibitors of the \ac{NSP13} helicase which can then be further investigated in experimental laboratory settings. This process involves several steps: (i) identification of potential drug binding sites, (ii) high-throughput screening of ligands to assess their binding efficacy within the  designated pocket, and finally (iii) evaluation of the binding pathways, kinetics, and thermodynamics \cite{Sledz_2018}. Focussing this screening on well-documented or already FDA-approved compounds is very attractive, as drug repurposing has the potential to shorten the development period and therefore also the development costs of new therapeutics \cite{Pushpakom_2019}.

%To explore new therapy options it makes sense to perform a  \textit{in-silico} screening of different compounds to identify groups that interact well with the \ac{NSP13} helicase. This screening process involves several steps: Identify possible binding sites, screen ligands for how well they bind in this pocket and validate the findings to generate a suggestion, as to which compounds would be worth it to investigate in a wet-lab setting \cite{Sledz_2018}. Focussing this screening on well documented or already FDA approved compounds significantly simplifies a potential registration of a potential new drug entering the market. 

\subsection{Identification of Consensus Binding Pocket}
In drug discovery, the initial step is to investigate the protein structure in order to analyse potential binding sites. These binding sites are cavities on the surface or the interior of the protein with suitable properties to bind a ligand. The functionality of a binding pocket is determined by its shape and location, but also by the amino acid residues which define its physico-chemical characteristics \cite{Stank_2016}. 
Different experimental and theoretical procedures exist to analyse the druggability of such binding pockets. Overall, merging distinct approaches enables a more precise prediction of the resulting consensus binding pocket which can then be further investigated using molecular docking \cite{Ricci_2022}. 


\subsection{Molecular Docking}
Molecular Docking programs are used to evaluate binding affinities between a potential drug candidate and the target protein. A key aspect of this task is the prediction of the ligand position, orientation, and conformation. Search-based methods approach this task by iteratively modifying the ligand pose. A scoring function summarises the likelihood and quality of each pose by predicting the binding free energy and consequently the pose with the best score is chosen. Among the most widely used tools are \textit{AutoDock Vina} \cite{Trott.2010} and \textit{Glide} \cite{Halgren.2004}, which mainly differ in their scoring functions. However, such search-based methods are computationally expensive. Therefore, in order to be able to screen large datasets, search-based methods are generally restricted to a previously defined binding pocket \cite{Corso.2022}. Consequently, potential other binding sites of a ligand are not assessed. Machine learning-based blind docking approaches try to address that problem by stochastically predicting the binding pocket and ligand pose based on learned characteristics and aligning them to each other. The most promising results are achieved by using \textit{Diffdock} \cite{Corso.2022}, a generative model which applies a reverse diffusion process to the docking paradigm. In this manner, \textit{Diffdock} iteratively transforms an uninformed noisy distribution over ligand poses defined by the degrees of freedom involved in docking (position, turns around its centre of mass, and twists of torsion angles) into a learned model distribution \cite{Corso.2022}. \citeauthor{Corso.2022} thereby describe this process as a progressive refinement of random ligand poses via updates of their translations, rotations and torsion angles.

%\subsection{Lead Drug Enhancement}
%In order to enhance the binding affinity of our drug candidates and thus their performance, we used AutoGrow4 (Version 4.0.3) \cite{package_Autogrow4} to generate novel compounds. Starting with the best binding compounds of our initial docking simulation with AutoDock Vina as generation zero, multiple new structures are generated by combining sub-structures of the first generation or by passing them through a set of possible chemical reactions after converting them into their respective SMILES codes. All of the generated compounds are ranked by their binding affinity. After passing several filters, the best-performing compounds are used as the seed for the next generation. Using this algorithm, compounds are found, which show higher binding affinities than the first generation. As AutoGrow4 labels all new structures by the path by which they were obtained, we can also evaluate the synthesizability.  

\subsection{Estimation of Toxicity and Synthetic Accessibility}
In addition to determining the activity of novel drug candidates on the therapeutic target, predicition of toxic effects is an indispensible step in drug design to be able to assess the predicted risk vs. benefit ratio of the potential drug \cite{roncaglioni2013silico}. As conventional \textit{in vivo} animal tests are time-consuming, expensive, and ethically controversial, researchers nowadays tend to favour \textit{in silico} methods as they are significantly cheaper and faster than wet expirements and they allow for simultaneous evaluation of large numbers of potential drug candidates \cite{raies2016silico,roncaglioni2013silico}. Therefore, \textit{in silico} toxicity tests are routinely integrated into the early stages of drug discovery in an attempt to minimise late-stage failures in drug design \cite{dearden2003silico}. Moreover, novel drugs must not only ensure the safety of patients but also have the capability for large-scale synthesis in order to one day be commercially viable. For that reason, determination of the synthetic accessibility, that is the ease of synthesis of a chemical compound, is essential for estimating the feasibility of an active compound as a pharmaceutical \cite{boda2007structure}. Therefore, in our pipeline we followed up the identification of the lead compounds that exhibit optimal binding affinity within the consensus pocket with an evaluation of the general toxicity and synthetic accessibility of these compounds to help estimate the suitability of the compounds as real-life pharmaceuticals against \ac{SARS-CoV-2}.

\subsection{Molecular Dynamics Simulation}
As the last step in our pipeline, a \ac{MD} simulation was be conducted using the best-scoring compounds as a ligand in the binding pocket of the protein. Simulating the movement of molecules in a system based on the attractive and repulsive forces between atoms helps with understanding the molecular interplay between the found ligands and the binding pocket. Whether or not the ligand stays inside the binding pocket throughout the whole simulation gives an estimation of how strongly it binds to the protein \cite{MD_Basics}. As this is the final analysis step, the result of this simulation estimates how the ligand would perform in clinical applications. Additionally, the frames generated by the software can be used to calculate different metrics regarding the binding strength, like the \ac{RMSF}, \ac{RMSD} and the binding affinities estimated by \ac{MM-PBSA}. The \ac{RMSD} measures the average displacement of the atoms throughout the simulation compared to the first frame of the simulation and estimates how much the protein moves and changes conformations over time. The \ac{RMSF} on the other hand calculates the movement of a certain atom or group of atoms over time compared to the average position of the atoms and groups \cite{RMSD_RMSF}. The \ac{MM-PBSA} is able to give an estimation of the binding affinity of the simulated protein-ligand pair \cite{MM_PBSA}.


\section{Material and Methods}
\subsection{Datasets from ZINC20 and ECBD}
A total of 1616 \ac{FDA}-approved drugs were downloaded in \textit{.sdf} format from the ZINC database \cite{Irwin.2020}. Additionally, 5016 files of the \ac{ECBD} pilot library were downloaded.

\subsection{Receptor and Ligand Preparation}
In this project we analysed crystal structures of the \ac{SARS-CoV-2} \ac{NSP13} helicase which were obtained by \cite{NSP13_basics} in a crystallographic fragment screening. Thus, the three protein structures (PDB codes: 6ZSL, 5RME, 5RM2) were downloaded from the \ac{RCSB PDB}. All three structures were used to determine a consensus binding pocket. As 6ZSL is the crystal structure with the highest resolution of 1.94 {\AA} and the other two structures show the \ac{NSP13} helicase complexed with a different fragment, we selected 6ZSL for further analysis in our drug discovery pipeline. Although the helicase is a homodimer, it was found that only the monomer is the catalytically active form \cite{Berta_2021}. Therefore, we concentrated our drug discovery only on the monomer of the helicase, namely on chain A. \\
Protein Data Bank (PDB) files often contain problems that first need to be resolved, so they can be used for further simulations. Therefore, we prepared the PDB protein structure using the \textit{PDBFixer} (Version 1.9) application, which among other things adds missing hydrogen and heavy atoms, builds missing loops and replaces non-canonical with canonical amino acids \cite{Eastman_2017}. Here, we added hydrogen atoms appropriate for the physiological pH of 7.4. The B chain of 6ZSL, identical to the A chain, contained in the crystallographic structure was removed as well as all phosphates and the ligands contained in the protein structures of 5RME and 5RM2. Additionally, for molecular docking using \textit{AutoDock Vina} %add citation here --> (Version 1.1.2, henceforth abbreviated as Vina, \textcite{Trott.2010}) but how should we handle the henceforth then?
and for the \ac{MD} simulation the zinc ions were removed. For consensus binding site detection 6ZSL was used as a reference structure to align 5RME and 5RM2 in \textit{PyMol} (Version 2.5.7, \textcite{PyMol_endnote}). This allowed better visualization and superimposition of the results.\\ 
Ligands were prepared using \textit{OpenBabel} (Version 3.1.1, \textcite{OpenBabel}) in order to convert implicit hydrogens into explicit hydrogens, to generate the necessary 3D structures of the ligands, and to split multi-molecule files into single ones. \textit{ADFR suite} (Version 1.0, \textcite{AutoDockFR}) was further used in order to convert all files into the \textit{.pdbqt} format, which is required by \textit{Autodock Vina} \cite{Trott.2010}. 

\subsection{Consensus binding site detection}
%To identify a potential ligand binding site of the NSP13 helicase, three different tools based on different methods were used on Chain A of the crystal structures 6ZSL, 5RM2, and 5RME: (i) \textit{Fpocket} (Version 3.0, \textcite{package_Fpocket}), (ii) \textit{PrankWeb 3} (accessed on... , \textcite{package_P2Rank, package_PrankWeb, package_PrankWeb3}), and (iii) \textit{FTMap} (accessed on..., \textcite{package_FTMAP}).  For \textit{Fpocket}, only pockets with a \ac{DScore} above 0.2 were considered. These were loaded into PyMol (Version 2.5.7,\textcite{PyMol_endnote}) together with the pockets detected with the other tools. Next, we focused only on the calculated pockets calculated by \textit{Fpocket} and \textit{PrankWeb} and identified overlapping pockets using the amino acid sequence and the visualization of the different pockets. These were then considered as one consensus binding site.Since, \textit{FTMAP} follows a docking approach of sixteen small probes rather than calculating pockets, this tool was used to validate that indeed clusters of the small molecules formed inside of the potential consensus binding pocket. Lastly, the coordinates of the consensus binding pocket needed for molecular docking were calculated using the public server at \textit{usegalaxy.org} of the Galaxy web platform \cite{galaxy}. The docking box size was set to $30 \si{\angstrom}^3$.
To identify a potential ligand binding site of the NSP13 helicase, three different tools based on different methods were jointly used on Chain A of the crystal structures 6ZSL, 5RM2, and 5RME: (i) \textit{Fpocket} (Version 3.0, \textcite{package_Fpocket}), (ii) \textit{PrankWeb 3} (accessed on 27.12.2023, \textcite{package_P2Rank, package_PrankWeb, package_PrankWeb3}), and (iii) \textit{FTMap} (accessed on 7.12.2023, \textcite{package_FTMAP}).  
\textit{Fpocket} utilizes a geometry-based algorithm based on Voronoi tessellation and sequential clustering to determine potential binding sites \cite{package_Fpocket}. Then, for each pocket different properties from the atoms of the pocket are calculated and a \ac{DScore} is assigned based on which the pockets are ranked \cite{package_Fpocket}. 
The \ac{DScore} is a weighted sum of the normalized mean local hydrophobic density, a polarity score %(sum of polarity over all amino acids in the pocket made up of a binary scheme with 0 for non-polar and 1 for polar), 
and a hydrophobicity score to lower the score of pockets where non druglike molecules could bind \cite{Ricci_2022}. In this project only pockets with a \ac{DScore} above 0.2 were considered. 
We also implemented \textit{PrankWeb 3} which is based on a machine-learning algorithm \textit{P2Rank}. It assigns structural, physico-chemical, and evolutionary features to points on the solvent accessible surface of a protein. From this information, the machine-learning model is built and used to predict and rank potential ligand binding sites based on a calculated cumulative ligandability score. It also calculates a probability for each pocket which is based on the confidence of the prediction.
The results of both tools were visualized in \textit{PyMol}. Then, overlapping hot spots were merged based on the overlapping amino acid sequence and by means of visual inspection.
The third tool we used is \textit{FTMap}. It uses docking results of sixteen small molecules differing in polarity, shape and size to identify binding hot spots with a fast Fourier transform correlation. The most favourable docked confirmations are determined by energy minimization and clustering. Since there is no relatable druggability scoring function as found in both the other tools, we used \textit{FTMap} for validation to see if indeed clusters of the small molecules formed inside the potential consensus binding pocket.
Lastly, the centre coordinates of the consensus binding pocket needed for molecular docking were calculated using the public server at usegalaxy.org of the Galaxy web platform \cite{galaxy}. The docking box size was set to have an edge length of 30 {\AA} to ensure that all amino acids of the pocket that could potentially interact with a ligand are considered and that larger ligands are included in the molecular docking simulation.

\subsection{Molecular Docking}
The molecular docking was performed twice. 
% First as an initial screening of all 5092 prepared ligands (1472 from the ZINC database, 3620 from the ECBD database).
% ECBD pdbqt: 3813 (problematic: 193) = 3620 (size: 28) = 3592
% ZINC pdbqt: 1567 (problematic: 95) = 1472 (size: 44) = 1428
For the first step \textit{AutoDock Vina} (Version 1.1.2, henceforth abbreviated as Vina, \textcite{Trott.2010}) was utilized. 
%As the receptor the monomer of the SARS-CoV-2 helicase (ID: 6ZSL) was used which includes only chain A. 
%Especially for Vina the zinc ions were removed and the resulting structure was converted into the \textit{.pdbqt} format through \textit{AutoDockTools} (Version 1.5.7 \cite{Goodsell2021}). 
The consensus pocket was introduced as the grid box with lengths of 30 \AA. The exhaustiveness was set to 30 and the maximum number of binding modes to 9. Taking advantage of multithreading, \ac{Vina} used the 28 CPUs accessible on the multi-core server \cite{Che2023}. A filter was applied on the set of ligands assuring only 3D structures smaller than the specified grid box were screened against the receptor 
%(1428 from the ZINC database and 3592 from the ECBD database). 
The filter was implemented in Python (Version 3.11.6, \textcite{Python}) and executed together with the Vina command in Bash script. The resulting nine different conformations for each ligand were ranked by their binding affinity and only the best value was considered in further steps. A number of ligands were later found to have multiple docking results due to an overlap between the two datasets and in accordance with previous steps only the best score was kept. The remaining 4863 ligands were ranked by their affinity score and the top one hundred were selected for next steps. \\
A second molecular docking was performed with those top scorers from the screening as well as \ac{ADP} and \ac{ATP}. The docking software \textit{Glide} provided by \textit{Schrödinger Inc.} \cite{Friesner2004} was accessed through \textit{Maestro} (Version 2022.3, \textcite{Maestro2022}). The included tools \textit{Protein Preparation Wizard} and \textit{LigPrep} \cite{Madhavi2013} were utilized to prepare the monomer helicase and ligands for the docking process with the OPLS4 force field \cite{Lu2021}. The zinc ions from the original protein structure were retained. The pH value was set to 7.0. Depending on the initial structure the ligand preparation generates a varying amount of conformations. In the analysis of the results only the best-performing conformation was included. The \textit{Receptor Grid Generation} tool was used to generate the receptor grid with the same binding pocket as in Vina. The docking with Glide was performed at standard precision mode and with flexibility of the ligands enabled \cite{Halgren.2004}. The criteria for the selection of the best-performing ligands was chosen to be the docking score. The interactions between the top scoring ligands and the receptor were documented. 

\subsection{Estimation of Toxicity and Synthetic Accessibility}
The general toxicity and synthetic accessibility of the given compounds was estimated using the machine-learning tool \textit{eToxPred} \cite{pu2019toxpred}. The SMILES files of the Top100 compounds from \textit{AutoDock Vina} \cite{Trott.2010} served as input for the pre-trained model. The toxicity predictor was pre-trained on the FDA-approved and the KEGG-drug datasets whose compounds were considered non-toxic as well as the TOXNET and the T3DB datasets whose compounds were considered toxic using a deep-belief-network based model. This predictor yields a Tox-score between 0 and 1 and in accordance to the paper, all compounds with a Tox-score below 0.58 were deemed non-toxic. The synthetic accessibility was reflected in a \ac{SAscore} which was obtained by training an extra-trees-based classifier on NuBBE, UNPD, FDA-approved, and DUD-E-active datasets.
 
\subsection{Validation of the binding-site for the Top 100 scorer}
The likelihood of our Top100 scorer binding within our consensus pocket was validated by blind docking via \textit{Diffdock} \cite{Corso.2022}. Thereby, all required sdf files were transformed into SMILES using \textit{OpenBabel} (Version 3.1.1, \textcite{OpenBabel}). As receptor, the prepared 6ZSL PDB files were utilized. \textit{Diffdock} was performed using the pre-trained scoring model\footnote{https://github.com/gcorso/DiffDock/tree/bc6b5151457ea5304ee69779d92de0fded599a2c/workdir/paper\_score\_model} to infer ligand conformations and ranking based on the included confidence model\footnote{https://github.com/gcorso/DiffDock/tree/bc6b5151457ea5304ee69779d92de0fded599a2c/workdir/paper\_confidence\_model}. The proposed default settings were used for the inference. These contained 20 inference steps, the generation of 40 samples per ligand, a batch-size of 10 and 18 actual denoising steps, whereby no noise was used for the final step of the reverse diffusion process. For the analysis of the \textit{Diffdock} output, the maximum and minimum coordinates for all samples generated per ligand were extracted using \textit{PyMOL} (Version 2.5.5, \textcite{PyMOL}) and aligned with the coordinates of our consensus pocket in Python (Version 3.10, \textcite{Python}). Based on that, the mean number of ligands within our binding site was calculated and set into relation to the rank posed on the ligand samples by the \textit{Diffdock} confidence score.

\subsection{Molecular Dynamics Simulation}
In order to validate the binding of the discovered compounds, we used \textit{GROMACS} (Version 2023.3, \textcite{packageGROMACS}) to simulate the drugs inside the consensus binding site of NSP13. To do so, the Protein-Ligand Complex tutorial by \Citeauthor{Lemkul2018} was followed \cite{Lemkul2018}. The a99SB-disp forcefield was used, as it was shown to recreate protein structures in different environments very accurately \cite{Forcefield}. The 6ZSL A-chain PDB file was used as input. 
As the ligands feature bonds and atoms not commonly seen in proteins, it is necessary to create a fitting force-field for them. For this, \textit{acpype} \cite{acpype} was used. The output files from \textit{Maestro Glide} were converted to PDB and then piped into \textit{acpype} which creates files that enable us to add the compound to the simulation environment. The output was then combined with the \textit{GROMACS} topology and \textit{.gro} files manually, according to \Citeauthor{Lemkul2018}.
Following the tutorial, the simulation space was set up and filled with water (a99SBdisp\_water used as its force-field). Sodium and chloride ions were added to create a net-zero charge system. Energy-minimization and \textit{NVT} / \textit{NPT} equilibration were conducted. This ensures a somewhat relaxed starting point for the simulation. The production run was set to simulate 100 ns. The \textit{.mdp} file containing the simulation parameters used for the final production run can be seen in the GitHub repository. Additionally, a second \ac{MD} simulation was run, using ANP as the ligand, in order to contextualize the result of the first simulation.\\
For the analysis of the simulation, \textit{GROMACS} internal tools were used to calculate the RMSD and RMSF of the protein, according to the \textit{GROMACS} manual \cite{packageGROMACS}. These scores give us insight into the movement and conformational changes during the simulation regarding protein and ligand. Furthermore, the simulation frames were extracted and rendered into a video using \textit{VMD} \cite{VMD}. For MM-PBSA calculations we used \textit{gmx\_MMPBSA} \cite{MMPBSA1}. This package enables direct estimation of binding affinities using the \textit{GROMACS} output files.

\section{Results}

\subsection{Consensus binding site detection}
%Before we could screen ligand libraries for potential inhibitors, the druggability of the surface of the NSP13 helicase needed to be investigated to determine ligand binding sites. For a more precise identification three different crystal structures (PDB codes: 6ZSL, 5RM2, 5RME) were analysed using three different tools which follow distinct computational methods. The calculated binding hot spots were visualised in \textit{PyMol} and overlapping ones were merged through visual examination to obtain a consensus binding site.
The first step of our drug discovery pipeline, is the detection of the consensus binding site using three distinct computational tools. 
Overall, the hot spots calculated with \textit{Fpocket} overlapping between the three crystal structures all ranked first among the calculated binding sites for each crystal structure. Their \acp{DScore} were 0.647 for 6ZSL, 0.383 for 5RM2, and 0.875 for 5RME. Also, the respective overlapping binding sites calculated with \textit{PrankWeb} ranked either first (5RM2 and 5RME) or second (6ZSL) among the calculated binding sites. Therefore, the respective \textit{Scores} and calculated \textit{Probability} was also among the highest for these binding sites (Score of 9.69 and Probability of 0.562 for 6ZSL, Score of 22.15 and Probability of 0.860 for 5RM2, and a Score of 17.91 and a Probability of 0809 for 5RME). Additionally, four hot spots were detected for 6ZSL and 5RME which contained some amino acids in common with the just mentioned binding site. These were also merged into the consensus binding pocket. However, they were ranked lower by \textit{PrankWeb} and also contained additional amino acids not present in the highest ranked binding sites. 
Interestingly, for the crystal structure 5RME a second binding hot spot was detected in proximity to the 1B domain. Nevertheless, because this binding site was only found in one crystal structure and the underlying pockets were ranked very low by both \textit{Fpocket} and \textit{PrankWeb}, this binding site was not considered for further analysis.
The resulting consensus binding pocket is located in the ATP binding pocket in between the two RecA-like domains A1 and A2 (Figure \ref{consensus_binding_pocket}). 
\begin{figure}
    \centering
    \includegraphics[width=0.3\textwidth]{consens_domain_big.pdf}
    \caption{Consensus binding pocket of the NSP13 helicase. The binding pocket is located in the ATP binding pocket in between the two RecA-like domains A1 and A2.}
    \label{consensus_binding_pocket}
\end{figure}
\FloatBarrier
%figure wird noch kleiner, wenn bild von pocket hinzugefügt wird

\subsection{Evaluation of Molecular Docking}
The 100 top scorers from Vina were subjected to Glide. The lead compounds as per Glide are listed in \autoref{tab:top_docking_scores}. All of them had a higher docking score and therefore higher affinity to the receptor than ADP, but a lower affinity compared to ATP.\\
In comparison, Vina and Glide results showed no correlation between their ranks (\autoref{fig:comp_glide_vina}). The Pearson correlation coefficient r of the scores is 0.13 with a p-value of 0.19 which is similar to the r value of the ranks at 0.12 with a p-value of 0.22. However, 3 of Vina's top 5 scorers (rank 1, 2 and 4) were still part of the Top 10 Glide results (rank 9, 10 and 8). 
Analysis of intermolecular interactions between ligand and receptor binding pocket showed many bonds our Top 1 scorer from Glide has in common with ADP and ATP (LYS 288, LYS 320 and ARG 443). LYS 288 was thereby of particular interest, as it further interacts with our Top 2 and Top 3 scorer (\autoref{tab:ligand_interactions}). In most cases these bonds are either hydrogen bonds or salt bridges with some exceptions like an aromatic hydrogen bonds, halogen bonds or pi-cation.
%
%When comparing the overall positions of EOS100380 (\autoref{fig:docking_EOS100380}) and ADP (\autoref{fig:docking_ADP}) within the binding pocket the similarities are even more apparent. 

%\subsection{Comparison of AutoDock Vina results}
%%This part only makes sense if they actually used the same pocket? Don't think they used the same 6zsl structure ...
%The Team Sorbonne5 also used AutoDock Vina for the docking of ligands from the pilot library of the ECBD database. Most of these ligands were also included in the former mentioned analysis. With the exception of two it is possible to map the top 10 ligands from Sorbonne to results of the analysis. Only one of their top scorers is included in the 100 preselected ligands. Their first ranked ligand with the affinity score -11.1 has a corresponding rank of 63. The remaining 7 were mapped to lower ranks starting with 123 and ending with up to 2626.

\subsection{Validation of binding-site for the Top10 scorer of Glide}
As shown in \autoref{pymol_Top3}, \textit{Diffdock} did not validate our binding site for the Top 1 and Top 2 Glide results. None of the predicted conformations were found within our binding pocket. The Top 3 Glide result had two of \textit{Diffdocks} best ranked conformations within our binding pocket. Furthermore, \textit{Diffdocks} confidence into conformations of our Glide Top 3 ligand seems to correlate with their position within our binding pocket (\autoref{Diffdock_plot}). This supports the assumption that the preferred binding site for this ligand is within our binding pocket. Among our Top 10 ligands from \textit{Glide}, better results according to \textit{Diffdock} were only achieved for our Top 5 ligand from \textit{Glide}.
\begin{figure}[h]
	\centering
	\captionsetup[subfigure]{skip=-15pt,position=top,labelfont=bf,labelformat=parens,singlelinecheck=false}
	\subcaptionbox{%
		\label{subfig:top1_glide_diffdock}}{\includegraphics[scale=0.24]{Top1_Glide_Rank5}}
	\subcaptionbox{%
		\label{subfig:top2_glide_diffdock}}{\includegraphics[scale=0.24]{Top2_Glide_Rank5}}
  	\subcaptionbox{%
		\label{subfig:top3_glide_diffdock}}{\includegraphics[scale=0.24]{Top3_Glide_Rank5}}
	\caption{Diffdock results for our Top 3 Glide ligands. Shown are always the 5 ligand conformations Diffdock was most confident in. (a) Glide Top-1 (b) Glide Top-2 and (c) Glide Top-3.}\label{pymol_Top3}
\end{figure}

\subsection{Top 100 Compounds Exhibit Low Toxicity and High Synthetic Accessibility}
After identifying the top 100 best-binding ligands through \textit{AutoDock Vina}, our subsequent analysis focused on evaluating their practical applicability as potential drugs by considering their predicted toxicity and synthetic accessibility. The resulting SAscore and Tox-Score for each compound were visualized in a scatter plot as seen in {Figure \ref{eToxPred}}. 

\begin{figure}[h]
    \begin{center}
      \includegraphics[width=0.78\textwidth]{etoxpred_result.pdf}
    \end{center}
    \caption{\textbf{Scatter plot of predicted toxicity and synthetic accessibility of the 100 best-binding compounds.} The predicted \ac{SAscore} was plotted against the Tox-Score for the top 100 best scorers from \textit{AutoDock Vina}. The top 3 scorers from \textit{Glide} are highlighted in red. The vertical grey line represents the threshold for the toxicity, with the compounds to the right of this line being considered non-toxic.}\label{eToxPred}
  \end{figure}

\noindent Of those 100 compounds, 99 presented with a Tox-score below the threshold of toxicity, indicating a low probability of being toxic to humans. The overall median Tox-Score is 0.24, with a mean of 0.26 across all compounds. Across all compounds the median SAscore of 2.69 and a mean of 2.87 which suggests that they are generally easy to synthesize. The top scorer from Glide, ZINC000096077632, was predicted to have a Tox-Score of 0.14 and an SAscore of 5.02.


\subsection{Molecular Dynamics Simulation Validates Binding of Top Scorer}
The \ac{MD} simulation is integral to validate the results of our pipeline. After the production run was finished, the resulting trajectory file was centred on the protein and modified to remove any ghosting and splitting of the protein at the borders of the simulation box. The last frame of the simulation was extracted and visualized using \textit{pyMOL} \cite{PyMOL}, which can be seen in Figure \ref{MD.Annotated}.
\begin{figure}[h]
  \begin{center}
    \includegraphics[width=0.65\textwidth]{last_frame_render_annotated.pdf}
  \end{center}
  \caption{\textbf{Visualization of EOS100380 inside the binding pocket at the end of 100 ns simulation.} The polar interactions are marked with yellow dotted lines. The amino acids involved in these interactions are labelled.}\label{MD.Annotated}
\end{figure}
The figure clearly demonstrated, that the top scoring compound of our Glide screening stayed inside the binding pocket until the end of the simulation. This was also confirmed by manual inspection of the frames generated in the simulation, which were combined to a video file using \textit{VMD}. Furthermore, the polar interactions between the ligand and residues in its proximity are shown. It seems, that EOS100380 is bound to the protein by interactions with Glu319, Lys320 and Arg442 of the NSP13 helicase. These interactions could also be validated using \textit{LigPlot+} \cite{LigPlot}.\\
To investigate the dynamics of these interactions between ligand and protein, the \ac{RMSD} and \ac{RMSF} were calculated, which can be observed in Figure~\ref{rms}. 
\begin{figure}[h]
  \begin{center}
    \includegraphics[width=1.0\textwidth]{RMSD_RMSF_combined.pdf}
  \end{center}
  \caption{\textbf{RMSD of the protein backbone and ligand (A) and RMSF of the protein backbone per residue during the simulation (B).} The RMSD was plotted over time for the whole 100 ns simulation. The respective \acp{RMSD} are shown in red (ligand) and blue (protein) The RMSF is shown per residue in the residue. The colour bar annotates the protein domain the respective residue belongs to, according to \textcite{Domains}. ZBD = Zinc Binding Domain}
  \label{rms}
\end{figure}
The \ac{RMSD} observed over the course of the simulation (Figure~\ref{rms}A) rises at first, but reaches a plateau in both the ligand and the protein. The RMSD seems to settle faster in the ligand, which exhibits less movement in general. Investigation into the RMSF per residue (Figure~\ref{rms}B) suggests, that some residues move significantly more than others. The highly mobile residues seem to be located in the zinc binding domain and domains 1B and 2A. The latter two play a role in the binding of the ligand. Thus, movement in these parts of the protein was to be expected. All in all, the protein and ligand seems to undergo conformational changes, however the system does not show major signs of instability. The ligand seems to settle in a somewhat stable conformation after roughly 50ns. More information on the movement of the protein can be seen when investigating the radius of gyration (Supplementary Figure \ref{gyr}). The estimation of the interaction energy by \ac{MM-PBSA} returned a $\Delta G_{solv} = -9.98 \frac{\textrm{kcal}}{\textrm{mol}}$. The RMSD and RMSF of the simulation using ANP as the ligand can be seen in Supplementary Figure \ref{anp}. The latter system shows a generally lower RMSD and an earlier plateau compared to the first simulation. For the RMSF similarities between the runs can be observed, especially looking at the ZBD. 

\FloatBarrier

\section{Discussion and Outlook}

In this work, we identified a consensus binding pocket of the NSP13 helicase using three distinct computational tools. It is located between the RecA-like domains A1 and A2 in the ATP binding pocket. This is in agreement with the work by \textcite{Berta_2021} who identified structurally important conserved motifs in the ATP binding site of SARS-CoV-2 NSP13. Out of the 16 amino acids they depicted to interact with ATP 13 are located in the consensus binding pocket identified by us. We conclude, that the combination of the three binding site detection tools \textit{Fpocket}, \textit{PrankWeb}, and \textit{FTMap} is a suitable approach to identify a consensus binding pocket. 
However, the consensus binding pocket is not necessarily the best binding site for a potential inhibitor of the NSP13 helicase, since the natural ligand ATP is the primary energy currency of the human body, therefore it plays a crucial role in fundamental processes. Thus, inhibiting the ATP binding site selectively is challenging. If failed, this could lead to severe side effects. Therefore, it is important to consider other binding sites as well. %Quelle fehlt noch!! aber hab noch nichts gefunden :(( 


The Tox-Score predictor of \textit{e}ToxPred was trained using \ac{FDA}-approved dataset as non-toxic incidences. Consequently, the low mean Tox-Score of the tested compounds aligns with our expectation, considering that the compounds from the ZINC database are derived from an \ac{FDA}-approved dataset. The single toxic incidence we detected was derived from the ECBD database which was consists of \ac{FDA}-approved and non-\ac{FDA}-approved molecules.
As highlighted by \citeauthor{pu2019toxpred}, natural compounds typically exhibit higher \ac{SAscore} values compared to synthetic compounds due to their inherent complexity \cite{pu2019toxpred}. The relatively high \ac{SAscore} of ZINC000096077632 can be explained by the fact that ZINC000096077632 corresponds to angiotensin-(1-7) which is a naturally occurring compound with a crucial role in the \ac{RAS} \cite{santosangiotensin}. The analysis of natural compound datasets by \citeauthor{pu2019toxpred} revealed a bimodal distribution in the \ac{SAscore}, with peaks around 3 and 5. Furthermore, the very low Tox-Score of the top scorer can also be explained by the fact that it is a naturally occurring molecule in the human body. \\

\noindent The investigation into the \acf{MD} simulation further validated the viability of our pipeline. The RMSD of the backbone protein atoms (see Figure~\ref{rms}A) was slightly higher than anticipated, but the system stabilized over the duration of the simulation. Looking at the RMSF per residue (see Figure~\ref{rms}B) it is apparent, that certain domains of the protein lead to this increase in the RMSD. Especially the zinc binding domain shows a lot of movement. This was also observed throughout the simulation with ANP as the ligand (see \ref{anp}). Its RMSD was lower in general, which was to be expected as it is structurally very similar to ATP, however the result gives confidence in the stability of our system. When discussing the general movement of domains in such simulations one should keep in mind, that the NSP13 A-chain is part of a much bigger replication complex \cite{NSP13_basics} in an \textit{in-vitro} setting. The conformational changes seen in this analysis would be severely hindered by other components in the complex. The domains 1B and 2A were suspected to be involved in a lot of conformational changes, as they make up a large part of the interaction surface between the protein and the ligand. Investigating the RMSD of the ligand shows, that it too settles in a stable conformation during the simulation time span. The result of the \ac{MM-PBSA} calculation also suggests that interactions happen between the protein and our ligand, which can be seen by a negative $\Delta G_{solv}$.
All in all, the analysis offers confidence, that Angiotensin 1-7 binds to the NSP13 helicase and stays bound through an extended period of time. This fits the work of \textcite{angio}, who propose Angiotensin 1-7 as a new therapy to support the recovery from Covid-19. Next to its role in the Renin-Angiotensin-Aldosteron System, it seemingly could also lower the viral damage to the body by inhibiting NSP13.



\section{Limitations of the project}
Regarding the \acf{MD} simulation, the project was held back by the tight time schedule. Running simulations for other top scoring ligands would have helped immensely in validating our pipeline further and compare the results of the top scorer. Furthermore, running multiple runs of the same simulation, for example using replica exchange with dynamical scaling \cite{REDS}, would have also further deepened our confidence in the results, as \ac{MD} simulations in and of themselves are a very stochastic process and should always be estimated using replicas. For the analysis of the protein-ligand complex using \textit{gmxMMPBSA} the same calculation should have been conducted on the MD run using ANP as the ligand, which was not possible due to technical difficulties.Thus, the given $\Delta G_{solv}$ of our ligand is a little difficult to interpret without a comparison value.
Furthermore, the original plan to implement \textit{AutoGrow4} \cite{packageAutogrow4} in order to improve our lead drugs and generate novel compounds was hindered by technical problems, which could not be fixed with the limited time at our disposal. We believe, that the increase in diversity among the drug candidates would lead to a better final drug candidate. With the help of \textit{eToxPred}, as shown in this report, the new compounds could have been evaluated for their toxicity and a well binding, not too cytotoxic lead could have been presented. 

\pagebreak
\section{Supplementary Material}

\setcounter{figure}{0}
\renewcommand{\thefigure}{S\arabic{figure}}

\begin{figure}[htp]
	\centering
	\captionsetup[subfigure]{skip=-15pt,position=top,labelfont=bf,labelformat=parens,singlelinecheck=false}
	\subcaptionbox{%
		\label{subfig:comp_ranks}}{\includegraphics[scale=0.5]{Comparison_of_ranks.png}}
	\subcaptionbox{%
		\label{subfig:comp_scores}}{\includegraphics[scale=0.5]{Comparison_of_scores.png}}
	\caption{Comparison of Vina and Glide docking results. 100 ligands were compared in \subref{subfig:comp_ranks}) ranks and \subref{subfig:comp_scores}) docking scores. Dots in blue represent the top 5 ligands from Vina and in orange the top 5 from Glide. All the other top 100 ligands are marked grey.}\label{fig:comp_glide_vina}
\end{figure}\\

% for appendix
\begin{table}[htp]
	\centering
	\caption{Glide docking scores for the top 4 ligands, ATP and ADP.}\label{tab:top_docking_scores}
	\begin{tblr}{
			hline{1-2,12} = {-}{},
		}
		Title            & docking score \\
		ATP              & -7.991        \\
		EOS100380        & -7.573        \\
		ZINC000008101127 & -7.273        \\
		EOS100897        & -6.786        \\
		ZINC000150588351 & -6.655        \\
		ADP              & -6.63         \\   
	\end{tblr}
\end{table}

\begin{table}[htp]
	\centering
	\caption{List of intermolecular interactions between ligands and receptor protein.}\label{tab:ligand_interactions}
	\begin{tblr}{
			hline{1-2,16} = {1-3}{},
		}
		Ligand           & Bond type       & Amino acid                                                        &  \\
		ADP              & hydrogen bond   & {GLY 287, LYS 288, SER 289, HIE 290, \\LYS 320, ARG 443, SER 539} &  \\
		& aromatic h-bond & LYS 320                                                           &  \\
		& salt bridge    & LYS 288, LYS 320, ARG 443                                         &  \\
		& Pi-cation       & ARG 443                                                           &  \\
		ATP              & hydrogen bond   & {GLY 287, LYS 288, SER 289, HIE 290, \\ARG 443, GLH 375}          &  \\
		& salt bridge    & LYS 288, LYS 320, ARG 443                                         &  \\
		ZINC000096077632 & hydrogen bond   & {LYS 288, LYS 320, ASP 315, GLU 319, \\GLU 341, ARG 443, GLU 540} &  \\
		& salt bridge    & {LYS 288, LYS 320, ASP 315, GLU 319,~ \\ARG 443, GLU 540}         &  \\
		ZINC000008101127 & hydrogen bond   & ARG 178                                                           &  \\
		& aromatic h-bond & GLY 538                                                           &  \\
		& salt bridge    & LYS 288                                                           &  \\
		ZINC000035880991  & hydrogen bond   & ASP 534                                                           &  \\
		& halogen bond    & ASN 516, THR 532                                                  &  \\
		& Pi-cation       & LYS 288                                                           &  
	\end{tblr}
\end{table}

% for appendix
\begin{figure}[htp]
	\centering
	\captionsetup[subfigure]{skip=-20pt,position=top,labelfont=bf,labelformat=parens,singlelinecheck=false}
	\subcaptionbox{%
		\label{subfig:EOS100380}}{\includegraphics[scale=0.5]{docking_EOS100380.png}}
	\subcaptionbox{%
		\label{subfig:EOS100380_s}}{\includegraphics[scale=0.57]{docking_EOS100380_surface.png}}
	\caption{Molecular docking of the top scoring ligand with Glide. The result of the docking process with the top scoring ligand (ECBD ID: EOS100380) is shown with focus on \subref{subfig:EOS100380}) the specific interactions between ligand and receptor protein and \subref{subfig:EOS100380_s}) position within the binding pocket. The different interactions between the two molecules are coloured depending on type. Hydrogen bonds are shown as yellow and salt bridges as violet.}\label{fig:docking_EOS100380}
\end{figure}

% for appendix
\begin{figure}[htp]
	\centering
	\captionsetup[subfigure]{skip=-20pt,position=top,labelfont=bf,labelformat=parens,singlelinecheck=false}
	\subcaptionbox{%
		\label{subfig:ADP}}{\includegraphics[scale=0.45]{docking_adp.png}}
	\subcaptionbox{%
		\label{subfig:ADP_s}}{\includegraphics[scale=0.5]{docking_ADP_surface.png}}
	\caption{Molecular docking of the ADP with Glide. The result of the docking process with ADP is shown with focus on \subref{subfig:ADP}) the specific interactions between ligand and receptor protein and \subref{subfig:ADP_s}) position within the binding pocket. The different interactions between the two molecules are coloured depending on type. Hydrogen bonds are shown as yellow, salt bridges as violet, aromatic hydrogen bonds as blue and pi-cation as green.}\label{fig:docking_ADP}
\end{figure}


\begin{figure}[h]
  \begin{center}
    \includegraphics[width=0.7\textwidth]{gyration_protein.pdf}
  \end{center}
  \caption{Radius of Gyration of the NSP13 protein throught the production run featuring EOS100380.}\label{gyr}
\end{figure}

\begin{figure}[h]
  \begin{center}
    \includegraphics[width=0.85\textwidth]{rms_ANP.pdf}
  \end{center}
  \caption{RMSD and RMSF of Protein backbone for the simulation with ANP}\label{anp}
\end{figure}

\begin{figure}[h]
  \begin{center}
    \includegraphics[width=0.8\textwidth]{diffdock_percentage_graph}
  \end{center}
  \caption{Percentage of ligand conformation within our binding site in relation to the selection made by Diffdock}\label{Diffdock_plot}
\end{figure}


\pagebreak
\FloatBarrier
\renewcommand{\bibname}{References}  % damit Literatuverzeicnis mit "References" betitelt
\printbibliography


\end{document}
